crates/solidity/inputs/language/definition/04-statements/01-blocks/notes.md

```diff
@@ -1,7 +1,14 @@
 --8<-- "crates/solidity/inputs/language/snippets/under-construction.md"
 
-## Blocks
+## Unchecked Blocks
 
 By default, all arithmetic operations are checked for underflow or overflow, which mean that if the result of an
```

Comment:
I think this could be "Starting with 0.8.0, by default, ...".

Reply:
#537